schemas/common/b2b-lead.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Lead",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Lead specific attributes.",
+  "definitions": {
+    "b2b-lead": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Lead Id",
+          "description": "Lead id.",
+          "type": "string"
+        },
+        "xdm:converted": {
+          "title": "Converted Indicator",
+          "description": "Indicates whether the Lead was successfully converted or not.",
+          "type": "boolean"
+        },
+        "xdm:convertedAccountID": {
+          "title": "Converted Account ID",
+          "description": "Account Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedContactID": {
+          "title": "Converted Contact ID",
+          "description": "Contact Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedDate": {
+          "title": "Converted Date",
+          "description": "Date of conversion if the lead was successfully converted.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
```

Comment:
Is Jigsaw a product name?  if so, then this should be in the `jigsaw` namespace

Reply:
Its not a product. Its an identifier provided by data.com. Its named as it is. This property is coming from Salesforce.  We did not change the name.

Reply:
See previous comment then about proper namespacing...

Reply:
@lrosenthol is suggesting we introduce a standard "salesforce" namespace and have this field be "salesforce:jigsawContactID". That makes sense if this is truly the source of that field and it's salesforce which defines what it means. We use "xdm" if we are creating a new definition for that field.

Reply:
Given this field has a very specific reference to the "data.com" definition, we can namespace it as "dataDotCom:jigsawContactID"

Reply:
Will add the dataDotCom namespace.